modules/openapi-generator/src/main/resources/Java/libraries/vertx/api.mustache

```diff
@@ -14,6 +15,8 @@ public interface {{classname}} {
     {{#operation}}
     void {{operationId}}({{#allParams}}{{{dataType}}} {{paramName}}, {{/allParams}}Handler<AsyncResult<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}Void{{/returnType}}>> handler);
 
+    void {{operationId}}({{#allParams}}{{{dataType}}} {{paramName}}, {{/allParams}}ApiClient.AuthInfo authInfo, Handler<AsyncResult<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}Void{{/returnType}}>> handler);
```

Comment:
@jekkel Looks like a breaking change to me as the method signature has changed.

Reply:
Thanks for reviewing. No, it's not a breaking change as it just adds an overload with a changed signature. The previous signature is still available.